easybuild/easyconfigs/i/icc/README.md

```diff
@@ -0,0 +1,13 @@
+In order to use these configs you must first download the Intel Parallel Studio XE CPP file. NOT the full bundle! The file should look like this: parallel_studio_xe_2018_update2_composer_edition_for_cpp.tgz    
+
+Next you must specify the license file. There are three options:  
+* Place it in the following path: ~/licenses/intel/license.lic
+* Set an environment variable: `export INTEL_LICENSE_FILE=/path/to/file.lic`
+* Set an environment variable: `export LM_LICENSE_FILE=/path/to/file.lic`
+
+To install, you must past eb the sourcepath to find the download. There are a number of ways to do this assuming the file is in ~/EB_Downloads.  
```

Comment:
`you must past eb the sourcepath to find the download.` ->

```
you must configure EasyBuild to use the right `sourcepath` so it can find the manually downloaded file.
```

Reply:
I agree with the wording change. I will update the request.